stories/Modal/Modal.story.mdx

```diff
@@ -0,0 +1,82 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+
+import Modal from '../../components/Modal';
+
+import { Header } from '../shared';
+
+import { 
+  SimpleModal,
+  WithButtonActions,
+} from './Modal.stories.jsx';
+
+<Meta title="Modal" component={Modal} />
+
+<Header title="Modal">
+   Modals inform users about a task and can contain critical information,
+      require decisions, or involve multiple tasks.
+</Header>
+
+- [Importing and usage](#importing)
+- [API](#api)
+- [Examples](#examples)
+
+## <a id="importing"></a>Importing and usage
+
+```js
+import { Modal, Button } from '@catho/quantum';
+```
+
+```jsx
+...
+import { Modal, Button } from '@catho/quantum';
+
+export default function SimpleModal() {
+  const [openModal, setOpenModal] = useState(false);
+
+  return (
+    <>
+      <Button onClick={() => setOpenModal(true)}>Open Modal</Button>
+      {openModal && <Modal>Content...</Modal>}
+    </>
+  );
+}
+```
+
+<Canvas>
```

Comment:
Since the modal code is already being shown above, I find it necessary to remove it from the canvas code view